docs/download.html

```diff
@@ -0,0 +1,546 @@
+            Scientific Linux 7.5 or later
+          </li>
+        </ul> 
+        <p>
+          See the Release Notes for oVirt 4.2.7 and the Installation Guide.
+        
+          oVirt can be downloaded and installed two primary ways: Self-Hosted Engine or separate Engine and Hosts.
+        
+          For production installations, we recommend installing oVirt using the Self-Hosted Engine configuration. In this configuration, oVirt Engine and a Host are installed together with the Engine running as a Virtual Machine on that Host. This configuration is more resilent because the Engine Virtual Machine will be highly available (once a second Host is added). See the Self-Hosted Engine Guide for full installation instructions for this configuration.
+        
+          If you prefer to run oVirt Engine standalone on physical hardware, you can install oVirt Engine and Nodes / Hosts separately. See the oVirt Installation Guide for full installation instructions for this configuration.
+        </p> 
+    </section>
+
+    <section class="ov-layout-styles ov-content-styles">
+      <h2 class="ov-heading ov-heading--lg">Download and Install Self-Hosted Engine</h2>
+      <div>
+        <p>
+        For production installations, we recommend installing oVirt using the Self-Hosted Engine configuration. In this configuration, oVirt Engine and a Host are installed together with the Engine running as a Virtual Machine on that Host. This configuration is more resilent because the Engine Virtual Machine will be highly available (once a second Host is added). See the Self-Hosted Engine Guide for full installation instructions for this configuration.
+
+        oVirt Self-Hosted Engine is installed using a graphical installer in Cockpit. Cockpit is enabled by default on oVirt Nodes (see below). If you are using a Enterprise Linux Host, see Installing Cockpit on Enterprise Linux Hosts in the Installation Guide.
+        </p>  
+      </div>  
+    </section>
+
+    <section class="ov-layout-styles ov-content-styles">
+      <h2 class="ov-heading ov-heading--lg">Red Hat Enterprise Linux, CentOS Linux</h2>
+      <div>
+       <ol>
+         <li> Enable the Base, Optional, and Extra repositories (they are enabled by default on CentOS). </li>
+         On Red Hat Enterprise Linux you'll need a valid subscription and the following repositories enabled:
+         <ul>
+            <li>rhel-7-server-rpms</li>
+            <li>rhel-7-server-optional-rpms</li>
+            <li>rhel-7-server-extras-rpms</li>
+          </ul>
+         <li>Add the official oVirt repository.</li> 
+         (add code snippet)
+         <li>Enable Cockpit.</li> 
+         (add code snippet)
+         <li>Install cockpit-ovirt-dashboard.</li> 
+         (add code snippet)
+       </ol>
+      </div>  
+    </section>
+
+    <section class="ov-layout-styles ov-content-styles">
+      <h2 class="ov-heading ov-heading--lg">Install oVirt Self-Hosted Engine</h2>
+      <div>
```

Comment:
same here with the `div` I'd just go through the whole page and remove any div's that aren't doing anything.